### DISCUSSION TITLE
General Setup Discussion

### DISCUSSION THREAD

**Discussion by donsteponmyeezy** [formfunction-hq/formfunction-monorepo#2] [General]
Hey guys, is anyone else having trouble getting the environment setup? It seems that the package:

@formfunction-hq/formfunction-auction-house@0.0.5-alpha.146

Is non-existent in the NPM registry. Let me know if any of you have had more success than me in installing the dependencies and running the project.

Thanks

**Comment by daniel-barker** [comment]
I was able to get passed the issue you faced here by creating a .yarnrc.yml file in the root directory and adding 

'nodeLinker: node-modules
npmScopes:
  formfunction-hq:
    npmRegistryServer: "https://npm.pkg.github.com"
    npmAuthToken: "GITHUB_PERSONAL_ACCESS_TOKEN"'

You can create a GH Personal Access Token in gh by going to settings -> developer settings and generating a classic token with at least write:package and read:package flags checked. 

but of course that just opened the door for ten tons of other dependency issues, that I myself will be making a post about.
